specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/examples/ExportDatabaseUsingDatabaseExtensionsWithManagedIdentity.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Are the examples part of this PR? I do not see them being called from the swagger json files changed in this PR .

Reply:
Thanks for pointing it out. Just fixed it in the last iteration of this PR.